projects/example-app/.eslintrc.json

```diff
@@ -29,6 +29,7 @@
           }
         ],
         "@typescript-eslint/prefer-namespace-keyword": "error",
+        "@nrwl/nx/enforce-module-boundaries": "off",
```

Comment:
I disabled the rule here, because this was too big of a change and because I think we want our example app to use this "syntax".